src/main/java/org/folio/rs/domain/entity/AccessionQueueRecord.java

```diff
@@ -34,4 +34,34 @@
   private String instanceTitle;
 
   private String instanceAuthor;
+
+  private String instanceCreator;
+
+  private String instanceContributors;
+
+  private String publisher;
+
+  private String publishYear;
+
+  private String publishPlace;
+
+  private String volume;
+
+  private String enumeration;
+
+  private String chronology;
+
+  private String issn;
+
+  private String isbn;
```

Comment:
Is it make sense to save list of identifiers in the map? For example, instance on FOLIO side can have only ISSN or all the ISSN, ISBN, OCLC.

Reply:
In case the certain identifier is absence,  the database column value will be empty. So map usage is not mandatory.

Reply:
@alekGbuz , let's follow standard way for identifiers representation: https://github.com/folio-org/mod-inventory/blob/f70c37d435594e9a6d108e66b879448f466108b2/ramls/instance.json#L120.